### PR TITLE
add build.sh and run.sh

### PR DESCRIPTION
**Pull request by MrKingHH**
I have added the build.sh and run.sh. If uyou are free, can you help me to review these code?
@wangqing1218 @zmzhouXJTU

### CODE REVIEW

**Comment by zmzhouXJTU** [comment]
sorry，I have seen it just now . I will review the code as soon as possible

**Comment by wangqing1218** [comment]
I have not received the review request.
I will review it right now.
By the way, I suggest that you should set the reviewers for next PR.

File: SS/nbanews/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+#
+sudo apt install python3-pip
```
Comment (wangqing1218) [inline SS/nbanews/build.sh:3]: Maybe we should judge wether  it have installed yet on this machine, like this:
```shell
python = $(which python)
if [python=='python' or python=='python3'] then
    echo "python have installed."
else
   sudo apt install python
fi
```

File: SS/nbanews/run.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+rm *.txt
```
Comment (wangqing1218) [inline SS/nbanews/run.sh:2]: I think it's a bad idea to remove all files which are end of  the `.txt` postfix. 
We can write the '.txt' files in `xxx/build` folder that represent what the program output.
Then, our script can be written:
```shell
rm xxx/build/*.txt
```

**Review by wangqing1218 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**